src/ra_utility.h

```diff
@@ -214,6 +216,21 @@ _CONSTANT_FN& operator&=(_Inout_ Enum& a, _In_ Enum b) noexcept
     return (a = a & b);
 }
 } // namespace bitwise_ops
+
+namespace arith_ops {
+template<typename Enum, typename = std::enable_if_t<std::is_enum_v<Enum>>>
+_NODISCARD _CONSTANT_FN operator-(_In_ Enum a, _In_ std::underlying_type_t<Enum> b) noexcept
+{
+    return ra::itoe<Enum>(ra::etoi(a) - b);
+}
+
+template<typename Enum, typename = std::enable_if_t<std::is_enum_v<Enum>>>
+_CONSTANT_FN& operator-=(_Inout_ Enum& a, _In_ std::underlying_type_t<Enum> b) noexcept
+{
+    return (a = a - b);
+}
+} // namespace arith_ops
```

Comment:
Pretty simple, just cast around with valid input. wasn't sure if we needed more though. On that note, I wasn't sure we needed more bitwise operator overloads at the time I did them.